packages/backend/tsconfig.json

```diff
@@ -1,14 +1,13 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "module": "esnext",
+    "lib": ["ES2019"],
+    "module": "commonjs",
+    "target": "ES2019",
     "outDir": "dist",
     "rootDir": ".",
     "resolveJsonModule": true,
-    "baseUrl": ".",
-    "paths": {
-      "@/*": ["src/*"]
-    }
```

Comment:
For my own understanding: why was this done?

Reply:
I removed it for now as it is not properly supported / configured and was throwing some errors.

Reply:
ok, thanks for the info 😃